### ISSUE TITLE
bug(mat-date-range-input): JAWS-E goes to the end-date field first

### ISSUE THREAD

**Issue by mcbashie** [angular/components#24690]
### What are you trying to do?

Hi,

I am using a mat-date-range-input and the client is using JAWS-E (Next Edit Box) to go from input-field to input-field.
JAWS-E however first goes to the end date and then to the start date.

Thanks&Regards,
Nelleke

### What troubleshooting steps have you tried?

As I thought I was going something wrong, I had asked a question about it on stackoverflow first.
https://stackoverflow.com/questions/71321140/using-mat-date-range-input-but-the-jaws-reader-first-reads-the-end-date-with-jaw

I got a very useful comment that stated that the issue is caused by the 'aria-owns'.
As this is generated by Angular I am unable to fix this myself.

### Reproduction

Steps to reproduce:
1. Add a mat-date-range-input element with a mat-label in the component.
2. Use JAWS-E Next Edit Box to navigate.
3. JAWS will first read the end-date field instead of the start-date.

### Environment

- Angular: 13.3.0
- CDK/Material: 
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10

**Comment by mcbashie** [comment]
I added 'with a mat-label' in step 1 of the reproduction.

**Comment by zarend** [comment]
confirming that I can reproduce on the [Date range input forms integration
](https://material.angular.io/components/datepicker/overview#date-range-picker-forms) example.

Windows 10 Enterprise 20H2 | Chrome Version 101.0.4951.67 (Official Build) (64-bit) | JAWS version 2021.2103.174 ILM

**Comment by zarend** [comment]
This issue was fixed by #25457 🎉
